lib/travis/build/config.rb

```diff
@@ -141,7 +141,8 @@ def sc_data
         ).split(/,/),
         dump_backtrace: ENV.fetch(
           'TRAVIS_BUILD_DUMP_BACKTRACE', ENV.fetch('DUMP_BACKTRACE', 'false')
-        ).to_bool
+        ).to_bool,
+        network_available: ENV.fetch('TRAVIS_WAIT_FOR_NETWORK_CHECK', 'true').to_bool
```

Comment:
I propose to not mixing config names and env.
Better is to have similar names of config as env: wait_for_network_check
but then make change in lib/travis/build/appliances/wait_for_network.rb
`return unless Travis::Build.config. wait_for_network_check` 
Is to_bool available here?
I saw commit 4911976 that changed it in the past.